gemrb/core/System/VFS.h

```diff
@@ -44,6 +44,7 @@
 
 namespace GemRB {
 
+// FIXME: this actually needs custom char_traits when we want to be case insensitive
```

Comment:
Might be simpler to just explicitly do a case insensitive check where needed, since it's what, 1 place right now?

Reply:
Maybe, I truly don't know the best route. We had some places that were compared with `stricmp` before tho, so those  are broken now without changing that. We'll just have to see if they matter.

Reply:
If I'm reading the diff correctly there's only that one place that was in FindInDir where you left another FIXME.

Reply:
I probably reverted some of the others I had originally changed.

It might be the best thing to do would be change `path_t` to a wrapper that can do case insensitive comparisons when that option was set, but thats not something I want to address here.

Reply:
This can go now, right? We have only one str(n)icmp in VFS and that's the one for comparing on-disk state.